lib/service/Service.ts

```diff
@@ -119,23 +119,27 @@ class Service {
   public channelBalance = async (args: { currency: string }) => {
```

Comment:
Would be nice to get test coverage for this public function as well. I've got a boilerplate service spec available in my open channel branch:
https://github.com/ExchangeUnion/xud/pull/1044/files#diff-560e7ab3ce0b2935a5eb94ee3c9be6d7R1

Reply:
Is it ok if I add the test case in a separate PR after that PR is merged?

Reply:
yes, merging this is prio

Reply:
That's fine. Let's do that.